### ISSUE TITLE
Tests should not be installed

### ISSUE THREAD

**Issue by moubctez** [angr/angr#4432]
### Description

Currently all tests get installed in `site-packages`, which is wrong. `tests` directory must be excluded from installed modules:

```
--- setup.cfg.orig
+++ setup.cfg
@@ -75,6 +75,9 @@ angr =
 	py.typed
 	lib/*
 
+[options.packages.find]
+exclude = tests*
+
 [egg_info]
 tag_build = 
 tag_date = 0
```

### Steps to reproduce the bug

Build the package.

**Comment by moubctez** [comment]
Here it is :)
https://github.com/angr/angr/pull/4433

**Comment by twizmwazin** [comment]
Fixed with #4433
